mantidproject/mantid
# Stitching tab for ORNL SANS

**Description of work.**

Updates the Stitching tab for ORNL SANS to use QtPy and makes it visible in Workbench

**Report to:** @AndreiSavici 

**To test:**



- Open ORNL SANS interface and verify that the  stitching tab is present in both MantidPlot and MantidWorkbench 
- To test: open the tab and load the following files. Then for each pair, click on the overlap region button. Select y axis to be logarithmic. Move the range lines to have parallel lines in the selection region. Make sure the x axis is updated in the GUI. Repeat for the other pair. At the end, click on Auto Scale button.

[PorasilBmid_q_1D.txt](https://github.com/mantidproject/mantid/files/4922424/PorasilBmid_q_1D.txt)
[PorasilBlow_q_1D.txt](https://github.com/mantidproject/mantid/files/4922425/PorasilBlow_q_1D.txt)
[PorasilBhigh_q_1D.txt](https://github.com/mantidproject/mantid/files/4922426/PorasilBhigh_q_1D.txt)

Fixes #28784

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
